package.json

```diff
@@ -21,15 +21,16 @@
     "imagemin-gifsicle": "^5.2.0",
     "imagemin-mozjpeg": "^7.0.0",
     "imagemin-optipng": "^5.2.1",
-    "imagemin-pngquant": "^5.1.0",
+    "imagemin-pngquant": "^6.0.0",
     "imagemin-svgo": "^6.0.0",
     "imagemin-webp": "^4.1.0",
     "loader-utils": "^1.1.0",
-    "object-assign": "^4.1.1"
+    "object-assign": "^4.1.1",
+    "webpack": "^4.17.2"
```

Comment:
I'm ok with making this a devDependency

Reply:
Actually, should be a `peerDependency`, isn't? :thinking:

Reply:
Done!